ui/apps/platform/src/Containers/Compliance/Dashboard/useComplianceRunStatuses.ts

```diff
@@ -78,7 +57,6 @@ export function useComplianceRunStatuses(): UseComplianceRunStatusesResponse {
         onError,
         errorPolicy: 'all',
         notifyOnNetworkStatusChange: true,
-        fetchPolicy: 'no-cache',
```

Comment:
Since we are always polling, if the GQL cache is stale it will quickly be updated anyway. Removing this `fetchPolicy` allows for smoother syncing of this state across Compliance 1.0 and ConfigMgmt pages.